ztp/source-crs/ClusterLogForwarder.yaml

```diff
@@ -6,11 +6,11 @@ metadata:
   annotations:
     ran.openshift.io/ztp-deploy-wave: "10"
 spec:
-  outputs: $outputs
+  outputs:
```

Comment:
Is an empty value legal here or should this line simply be removed (or commented out)? The original policyGen would strip this line out if the user doesn't supply a value, so it seems like the equivalent would be to simply omit the line altogether (user can still provide it via overlay). I think this would ensure that the source-cr behaved in the same way for original policyGen and ACM version as well as still providing backwards compatible behavior for the original policyGen.

Same thing below for pipelines